GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -527,8 +527,8 @@ private void blameMenuItem_Click(object sender, EventArgs e)
                 return;
             }
 
+            int? line = FileText.Visible ? FileText.CurrentFileLine : BlameControl.CurrentFileLine;
             blameToolStripMenuItem1.Checked = !blameToolStripMenuItem1.Checked;
-            int? line = FileText.Visible ? FileText.CurrentFileLine : null;
```

Comment:
Why moved? `line` is given to the call below.

Reply:
I believe that `blameMenuItem_Click()` could change the line number